05-classification.Rmd

```diff
@@ -400,33 +415,18 @@ cancer %>%
   slice(1:5) # subset the first 5 rows
```

Comment:
The line with mutate above is too long and won't render well in LaTeX, change it to:

```
  mutate(dist_from_new = sqrt((Perimeter - new_obs_Perimeter)^2 + 
                                (Concavity - new_obs_Concavity)^2)) %>%
```

Reply:
done